test/e2e/runner/main.go

```diff
@@ -92,6 +98,8 @@ func NewCLI() *CLI {
 	cli.root.PersistentFlags().StringP("file", "f", "", "Testnet TOML manifest")
 	_ = cli.root.MarkPersistentFlagRequired("file")
 
+	cli.root.Flags().BoolVarP(&cli.preserveOutput, "preserve", "p", false, "Preserve node logs and state")
```

Comment:
I would probably just call this `preserve` or `keep` or something. It doesn't just preserve the logs and state, it keeps the testnet running indefinitely.

Reply:
Okay perhaps it's more desirable if we still halt the network but don't clean up afterwards.

The reason why I would like this is that even if all the tests pass I want to be able to examine the logs to make sure all the in intermediate steps happened properly and not just the result.

Reply:
I think keeping the network running is fine. Personally, I often want to inspect e.g. node RPC as well - I've typically just used `start` in these cases, but it's nice to be able to run the whole sequence first.